### PR TITLE
Removed warnings from the console

### LINKED ISSUE #2265: iframe does not have title
**Actual Behaviour**

iframe does not have a unique title. Many iframe errors are there after `npm start`.
Capture

**Expected Behaviour**

Providing a unique title to iframe

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2265 

Changes: Added title for every iframe tag (Removed warning after `npm start`) also removed warnings related to jsx-a11y/anchor-is-valid

Demo Link: https://pr-2273-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js [hunk 1]
```diff
@@ -92,7 +92,12 @@ const generateGifBubble = (
     <MessageContainer key={action + index}>
       <MessageBubble author={message.authorName}>
         <div>
-          <iframe src={gifSource} frameBorder="0" allowFullScreen />
+          <iframe
+            src={gifSource}
+            title="gifFrame"
```
Comment (akshatnitd) [inline src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js]: Here the title need not be camelCase. Write proper title, like `SUSI features GIF`

**Comment by samagragupta** [comment]
@akshatnitd Made all the required changes

File: src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js (hunk 1 above)
Comment (akshatnitd) [reply src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js]: @samagragupta This change was not made.

**Comment by samagragupta** [comment]
@akshatnitd Direclty using `#` will give a warning.

**Comment by samagragupta** [comment]
@akshatnitd Done the other changes

**Comment by akshatnitd** [comment]
@samagragupta What error is thrown when you directly write `#`?

**Comment by samagragupta** [comment]
@akshatnitd It is giving this error.
Capture

**Comment by akshatnitd** [comment]
@samagragupta Please resolve the PR soon. It is creating build fails.

**Comment by samagragupta** [comment]
@akshatnitd @fragm3 Changed anchor tag to div

**Comment by samagragupta** [comment]
@fragm3 Done

**Comment by samagragupta** [comment]
@akshatnitd I was waiting if changes made by me are fine. After you approve I would squash the commits.

**Comment by samagragupta** [comment]
@harsh-jindal Made the changes

**Comment by samagragupta** [comment]
@akshatnitd Done

**Comment by harsh-jindal** [comment]
@samagragupta Resolve conflicts

**Comment by samagragupta** [comment]
@harsh-jindal Resolved the conflicts

**Comment by samagragupta** [comment]
@harsh-jindal Changed the size.
Hope this size looks fine.

File: src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js [hunk 2]
```diff
@@ -92,7 +92,12 @@ const generateGifBubble = (
     <MessageContainer key={action + index}>
       <MessageBubble author={message.authorName}>
         <div>
-          <iframe src={gifSource} frameBorder="0" allowFullScreen />
+          <iframe
+            src={gifSource}
+            title="SUSI features GIF"
```
Comment (fragm3) [inline src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js]: SUSI GIF

File: src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js [hunk 3]
```diff
@@ -224,7 +229,12 @@ const generateAudioBubble = (
     <MessageContainer key={action + index}>
       <MessageBubble author={message.authorName}>
         <div>
-          <iframe src={src} frameBorder="0" allowFullScreen />
+          <iframe
+            src={src}
+            title="SUSI features audio"
```
Comment (fragm3) [inline src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js]: SUSI Audio

File: src/components/Dialog/Tour.js [hunk 4]
```diff
@@ -23,6 +23,7 @@ const Tour = props => {
         width="99%"
         height="315"
         src="https://www.youtube.com/embed/9T3iMoAUKYA"
+        title="SUSI tour guides"
```
Comment (fragm3) [inline src/components/Dialog/Tour.js]: SUSI Tour guide

File: src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js (hunk 2 above)
Comment (samagragupta) [reply src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js]: This title was suggested by Akshat Garg. And similarly others.

**Comment by harsh-jindal** [comment]
@samagragupta Fix this
Screenshot 2019-06-20 at 8 17 33 PM

**Comment by samagragupta** [comment]
@harsh-jindal Actually mobile view fixes are not in the scope of this PR. But I have made the changes.

**Comment by akshatnitd** [comment]
Resolve conflicts

**Comment by samagragupta** [comment]
@akshatnitd Resolved

**Pull request merged**